runtime/ops/http.rs

```diff
@@ -543,6 +557,10 @@ impl Resource for ResponseBodyResource {
   fn name(&self) -> Cow<str> {
     "responseBody".into()
   }
+
+  fn close(self: Rc<Self>) {
+    self.cancel.cancel()
```

Comment:
Why cancel a response?
Note that the general pattern that we use is to cancel read/accept/connect ops but not write ops.

Reply:
Ooops, looks like artifact from initial implementation. Removed!